hamster_lib/objects.py

```diff
@@ -477,7 +499,7 @@ def comma_split(string):
             category, description = None, None
 
         activity = Activity(activity_name, category=category)
-        return cls(activity, start, end=end, description=description)
+        return cls(activity, start, end=end, description=description, tags = tags)
```

Comment:
there should not be kwarg assignment should not be surrounded by whitespace as per [PEP8](https://www.python.org/dev/peps/pep-0008/#other-recommendations)